docs/api-reference/automatic-compaction-api.md

```diff
@@ -258,6 +258,88 @@ Host: http://ROUTER_IP:ROUTER_PORT
 
 A successful request returns an HTTP `200 OK` message code and an empty response body.
 
+### Update cluster-level compaction config
+
+Updates cluster-level configuration for compaction tasks which applies to all datasources, unless explicitly overridden in the datasource compaction config.
+This includes the following fields:
+
+|Config|Description|Default value|
+|------|-----------|-------------|
+|`compactionTaskSlotRatio`|Ratio of number of slots taken up by compaction tasks to the number of total task slots across all workers.|0.1|
+|`maxCompactionTaskSlots`|Maximum number of task slots that can be taken up by compaction tasks.|2147483647 (i.e. total task slots)|
+|`compactionPolicy`|Policy to prioritize intervals for compaction|`newestSegmentFirst`|
```

Comment:
Can the description list all the possible values for this property please?

Reply:
Also I noticed that the return of `GET /druid/coordinator/v1/config/compaction` has a slightly different shape.

<img width="279" alt="image" src="https://github.com/user-attachments/assets/dc786d8e-a140-4db4-a8f5-8a2bfc8fae93" />

What is `priorityDatasource`? Can the same shape of object be submitted?

Reply:
I tried posting

<img width="310" alt="image" src="https://github.com/user-attachments/assets/a50d0274-62e4-46ee-9ea9-cc118023e730" />

and got `"Could not resolve subtype of [simple type, class org.apache.druid.server.compaction.CompactionCandidateSearchPolicy]: missing type id property 'type' (for POJO property 'compactionPolicy')"` so maybe it HAS to be that shape.

Reply:
Yes, should have kept this PR in draft. Still working on the docs.
But yes, the result of the GET is the correct format.